### PR TITLE
JBIDE-27874: Fix PR Check failures

### PR DESCRIPTION
**Pull request by sbouchet**

### CODE REVIEW

**Comment by sbouchet** [comment]
likely close #480

**Review by jeffmaury — APPROVED** [review]
LGTM

File: .github/workflows/publish_reports.yml [hunk 1]
```diff
@@ -45,5 +45,5 @@ jobs:
         uses: scacap/action-surefire-report@v1
         with:
           github_token: ${{secrets.GITHUB_TOKEN}}
-          report_paths: '${{github.workspace}}/*test-reports/**/TEST-*.xml'
+          report_paths: '${{github.workspace}}/*test-reports/*tests/**/TEST-*.xml'
```
Comment (odockal) [inline .github/workflows/publish_reports.yml:48]: is it necessary?

**Review by odockal — CHANGES_REQUESTED** [review]

File: .github/workflows/publish_reports.yml (hunk 1 above)
Comment (sbouchet) [reply .github/workflows/publish_reports.yml:48]: it was part of your PR tentative that i started to work with. i can revert now that the tests are fixed.

Comment (odockal) [reply .github/workflows/publish_reports.yml]: cool, I was just curious, if there was a reason for it. Previous (without *tests) is more general to fit more components (all?).

**Review by odockal — APPROVED** [review]

File: .github/workflows/publish_reports.yml (hunk 1 above)
Comment (sbouchet) [reply .github/workflows/publish_reports.yml]: no reasons :)  you're right the previous is more general to catch up tests results.

**Pull request merged**